src/Faker/Provider/en_MY/Address.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace Faker\Provider\en_MY;
+
+class Address extends \Faker\Provider\Address
+{
+
+    // https://en.wikipedia.org/wiki/Addresses_in_Malaysia
+    protected static $streetNumber = array('##', '###', '####');
+
+
+    // http://www.streetdirectory.com/malaysia/kl/asia_travel/street/
+    protected static $streetSuffix = array(
+        
+    );
+
+    // http://www.streetdirectory.com/malaysia/kl/asia_travel/street/
+    protected static $streetPrefix = array(
+        'Jalan', 'Lorong',
+    );
+
+    // http://www.streetdirectory.com/malaysia/kl/asia_travel/street/
```

Comment:
The content of this site is copyrighted. Please find a datasource compatible with the MIT license.